### ISSUE TITLE
Get rid of redundant "Preferences" title in the preferences dialog

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#4652]
The title inside the window is redundant with the text in the titlebar and takes up a lot of precious vertical space:

![CleanShot 2023-05-11 at 10 07 40@2x](https://github.com/rancher-sandbox/rancher-desktop/assets/78947/eaed2ede-dbb2-4813-861e-e68a8c9f6f76)

Many apps also put the name of the currently selected preferences into the window title instead of the app name and just a static "Preferences" name, but that may be specific to macOS, where the app name is obvious from the title bar; on Windows and Linux the app name probably should be included. Examples:

![CleanShot 2023-05-11 at 10 10 19@2x](https://github.com/rancher-sandbox/rancher-desktop/assets/78947/9609585a-010c-4e31-9f93-49fe31cffbc4)

![CleanShot 2023-05-11 at 10 11 45@2x](https://github.com/rancher-sandbox/rancher-desktop/assets/78947/3645f77b-ed89-422a-ae5f-cadcd56743ac)
![CleanShot 2023-05-11 at 10 13 10@2x](https://github.com/rancher-sandbox/rancher-desktop/assets/78947/ba46bbe0-aa02-499e-bf43-85fe8b175f1c)

**Comment by jandubois** [comment]
I think the "Preferences" header inside the window is a left-over from the time when the preferences were still a page in the main window. I can't think of any reason to keep it.

As for the window title itself, I would use the current page name on macOS: `Application`, `Virtual Machine` etc.

On Windows and Linux we keep the prefix: `Rancher Desktop - Application` etc.

I've seen one example that did append "Preferences" at the end of the title: `Application Preferences`, `Rancher Desktop - WSL Preferences` etc. I don't see any reason to do so but wanted to list it here for completeness.

**Comment by manuelecarlini** [comment]
@jandubois I made some tests and I see these 3 possible variations.
I personally like it with Preference + Page title because being the panel on a separate window I'd still want to clarify that those are only the preference and not the application itself.

 
![Pagetitle-preference-Ranche destop](https://github.com/rancher-sandbox/rancher-desktop/assets/6459624/e0a02d7b-f1fe-49d9-a615-3f96bba56a40)

**Comment by kwwii** [comment]
I think that for the initial implementation should be the "No header" version shown above. That should be relatively easy to implement as we just remove the large preferences title of the app and set a static text to the app window "Rancher Desktop - Preferences". 

If we find that we need to add more specific information about the current menu selection we can add that in a separate step.

**Comment by jandubois** [comment]
> I think that for the initial implementation should be the "No header" version shown above.

Agreed. Changing the actual window title should be a separate issue. Sorry for mixing this into this story; it was mostly meant for context.

**Comment by jandubois** [comment]
Let's ignore the titlebar text for now and leave it as-is on all platforms.

This story is solely about removing the top box inside the window with the static "Preferences" string inside, so we get more vertical space for our content.
